models/surveys.go

```diff
@@ -212,6 +212,12 @@ func stripChars(str string, fn runevalidator) string {
 	}, str)
 }
 
+// IsValidUUID validate whether a string is a valid UUID
+func IsValidUUID(u string) bool {
```

Comment:
Do you need this function? Could just do it inline in the code (only used twice, don't need to over-DRY-out)
```
if _, err := uuid.FromString(u); err != nil {
    writeRestErrorResponse(w, "'"+surveyID+"' is not a valid UUID", http.StatusBadRequest)
    return
}
```

Reply:
I think it could theoretically be quite a useful helper method, though how much further this service is going to be expanded is debatable :D

Reply:
Doing it this way also swallows the error - doing it just inline (or at least returning the error) means it can be logged and dealt with at the appropriate place.